test/e2e/common_suite_test.go

```diff
@@ -345,15 +345,19 @@ func (tc *testCase) run() {
 				}
 				log.Infof("Deleting pod %s...", tc.pod.Name)
 
-				if tc.deletionWithin != nil {
-					if err = wait.For(conditions.New(
-						client.Resources()).ResourceDeleted(tc.pod),
-						wait.WithInterval(5*time.Second),
-						wait.WithTimeout(*tc.deletionWithin)); err != nil {
-						t.Fatal(err)
-					}
-					log.Infof("Pod %s has been successfully deleted within %.0fs", tc.pod.Name, tc.deletionWithin.Seconds())
+				// Add a default deleteWithin time.
+				if tc.deletionWithin == nil {
+					deleteWithin := time.Duration(time.Second * 300)
+					tc.deletionWithin = &deleteWithin
+				}
+
+				if err = wait.For(conditions.New(
+					client.Resources()).ResourceDeleted(tc.pod),
+					wait.WithInterval(5*time.Second),
+					wait.WithTimeout(*tc.deletionWithin)); err != nil {
+					t.Fatal(err)
 				}
+				log.Infof("Pod %s has been successfully deleted within %.0fs", tc.pod.Name, tc.deletionWithin.Seconds())
```

Comment:
I find that log line confusing. Would suggest to either print the real time the deletion took, or just remove the time part.

Reply:
done